packages/flutter/lib/src/painting/image_resolution.dart

```diff
@@ -116,7 +116,9 @@ const double _kLowDprLimit = 2.0;
 /// ```dart
 /// AssetImage('icons/heart.png')
 /// ```
-///
+/// 
+/// To understand how an [Image] object is made out of an [AssetImage], refer [ImageProvider] and [AssetBundleImageProvider].
```

Comment:
We'll have to adjust the Engilsh here for grammar and to fit the Flutter style, something more like:

```
/// For an example of resolving an [AssetImage] to obtain a [dart:ui.Image] object,
/// refer to the sample code in the documentation for [ImageProvider].
```